roxctl/central/generate/generate.go

```diff
@@ -203,6 +203,15 @@ func createBundle(logger logger.Logger, config renderer.Config) (*zip.Wrapper, e
 func OutputZip(logger logger.Logger, config renderer.Config) error {
 	logger.InfofLn("Generating deployment bundle...")
 
+	if config.EnablePodSecurityPolicies {
+		logger.InfofLn("Central deployment bundle includes PodSecurityPolicies (PSPs). This is incompatible with Kubernetes >= v1.25.")
+		logger.InfofLn("Use --enable-pod-security-policies=false for disabling PodSecurityPolicies.")
```

Comment:
I am not a grammar expert, but using this form sounds cleaner to me:

```suggestion
		logger.InfofLn("Use --enable-pod-security-policies=false to disable PodSecurityPolicies.")
```

Reply:
I think you are right.

Reply:
done